Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,15 @@
 import React from 'react';
 import './Header.css';
+import HeaderTitle from './HeaderTitle';
+import HeaderContent from './HeaderContent';
+import ImageThumbnail from './ImageThumbnail';
+const HeaderContainer = () =>{
+    return(
+       <div> <HeaderTitle /> 
```

Comment:
This is correct but I would recommend to fix your code layout by making a newline and indenting.